crates/modelardb_client/src/main.rs

```diff
@@ -214,56 +196,67 @@ fn repl(
 
 /// Execute an action, a command, or a query. Returns [`Error`] if the action, command, or query
 /// could not be executed or their result could not be retrieved.
-fn execute_and_print_action_command_or_query(
-    runtime: &Runtime,
+async fn execute_and_print_action_command_or_query(
     flight_service_client: &mut FlightServiceClient<Channel>,
     action_command_or_query: &str,
-) -> Result<(), Box<dyn Error>> {
-    if action_command_or_query.starts_with('\\') {
-        execute_command(runtime, flight_service_client, action_command_or_query)?;
-    } else if action_command_or_query.starts_with("SELECT")
+) {
+    let start_time = Instant::now();
+
+    let result = if action_command_or_query.starts_with('\\') {
+        execute_command(flight_service_client, action_command_or_query).await
+    } else if action_command_or_query.starts_with("INSERT")
```

Comment:
does this mean that "INSERT" works, but "insert" does not?

Reply:
Fixed through trim and uppercase.